fuel/app/classes/model/notification.php

```diff
@@ -153,45 +140,57 @@ public static function send_item_notification(int $from_user_id, int $to_user_id
 	 *
 	 * @param int ID of the notification to send an e-mail for.
 	 */
-	protected static function send_email_notifications()
+	public function send_email()
 	{
 		if ( ! \Config::get('materia.send_emails', true)) return;
 
 		\Package::load('email');
 		$email = \Email::forge();
 
-		$notes = \Model_Notification::query()
-			->where('is_email_sent', '0')
-			->limit(100)
-			->get();
+		$from = static::get_user_or_system($this->from_id);
+		$to = \Model_User::find($this->to_id);
+
+		$email->from(\Config::get('materia.system_email'), $from->first.' '.$from->last);
+		$email->reply_to($from->email, $from->first.' '.$from->last);
+		$email->to($to->email, $to->first.' '.$to->last);
+		$email->subject(\Config::get('materia.name').' Notification');
+		$email->html_body($this->subject);
+trace($from->email);
```

Comment:
Left some trace statements in here by accident?

Reply:
Gotem